pkg/cmd/operator/sidecar.go

```diff
@@ -95,6 +97,7 @@ func NewSidecarCmd(streams genericclioptions.IOStreams) *cobra.Command {
 	cmd.Flags().StringSliceVar(&o.ExternalSeeds, "external-seeds", o.ExternalSeeds, "The external seeds to propagate to ScyllaDB binary on startup as \"seeds\" parameter of seed-provider.")
 	cmd.Flags().StringVarP(&o.NodesBroadcastAddressTypeString, "nodes-broadcast-address-type", "", o.NodesBroadcastAddressTypeString, "Address type that is broadcasted for communication with other nodes.")
 	cmd.Flags().StringVarP(&o.ClientsBroadcastAddressTypeString, "clients-broadcast-address-type", "", o.ClientsBroadcastAddressTypeString, "Address type that is broadcasted for communication with clients.")
+	cmd.Flags().BoolVarP(&o.DeveloperMode, "developer-mode", "", o.DeveloperMode, "Specifies if ScyllaDB runs in developer mode.")
```

Comment:
Can we prepend this to the arbitrary scylladb args in the operator and avoid plumbing in an extra sidecar flag that will eventually do the same inside the sidecar?

Reply:
good idea, moved to positional arg